src/main/java/step02/code/domain/MakeNumber.java

```diff
@@ -0,0 +1,8 @@
+package step02.code.domain;
+
+import java.util.List;
+
+@FunctionalInterface
+public interface MakeNumber {
+  public List<Integer> make();
```

Comment:
```suggestion
  List<Integer> make();
```
인터페이스에서 `pulbic` 지시자는 생략할 수 있습니다.

Reply:
아.. 앞으로 인터페이스에는 지시자를 생략하도록 하겠습니다~